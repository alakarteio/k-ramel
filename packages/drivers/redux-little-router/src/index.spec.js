/* eslint-env jest */
import driver from './index'
import tests from '../misc/test-suite'

tests(driver)
