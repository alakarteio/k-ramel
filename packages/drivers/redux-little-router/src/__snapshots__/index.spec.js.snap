// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`drivers redux-little-router dispatch actions should dispatch block 1`] = `"ROUTER_BLOCK"`;

exports[`drivers redux-little-router dispatch actions should dispatch go 1`] = `"ROUTER_GO"`;

exports[`drivers redux-little-router dispatch actions should dispatch goBack 1`] = `"ROUTER_GO_BACK"`;

exports[`drivers redux-little-router dispatch actions should dispatch goForward 1`] = `"ROUTER_GO_FORWARD"`;

exports[`drivers redux-little-router dispatch actions should dispatch push 1`] = `"ROUTER_PUSH"`;

exports[`drivers redux-little-router dispatch actions should dispatch replace 1`] = `"ROUTER_REPLACE"`;

exports[`drivers redux-little-router dispatch actions should dispatch unblock 1`] = `"ROUTER_UNBLOCK"`;

exports[`drivers redux-little-router dispatch selectors should get router state 1`] = `
Object {
  "hash": "",
  "key": "6dwpw7",
  "options": Object {},
  "params": Object {},
  "pathname": "/foo",
  "previous": Object {
    "hash": "",
    "key": undefined,
    "params": Object {},
    "pathname": "/foo",
    "query": Object {},
    "queue": Array [
      Object {
        "hash": "",
        "options": Object {},
        "pathname": "/foo",
      },
    ],
    "result": Object {
      "parent": Object {
        "route": "/",
      },
      "title": "FOO_ROUTE",
    },
    "route": "/foo",
    "search": "",
    "state": undefined,
  },
  "query": Object {},
  "queue": Array [],
  "result": Object {
    "parent": Object {
      "route": "/",
    },
    "title": "FOO_ROUTE",
  },
  "route": "/foo",
  "routes": Object {
    "/": Object {},
    "/foo": Object {
      "parent": Object {
        "route": "/",
      },
      "title": "FOO_ROUTE",
    },
  },
  "search": "",
}
`;
